server/athenian/api/openapi/openapi.yaml

```diff
@@ -1056,3 +1087,167 @@ components:
       - date_from
       - date_to
       type: object
+    FilterPullRequestsRequest:
+      example:
+        account: 1
+        date_from: 2000-01-23
+        date_to: 2020-05-23
+        in:
+        - '{1}'
+        stages:
+        - wip
+        with:
+          author:
+          - github.com/vmarkovtsev
+      properties:
+        account:
+          description: Session account ID.
+          type: integer
+        date_from:
+          description: PRs must be updated later than or equal to this date.
+          format: date
+          type: string
+        date_to:
+          description: PRs must be updated earlier than or equal to this date.
+          format: date
+          type: string
+        in:
+          $ref: '#/components/schemas/RepositorySet'
+        stages:
+          items:
+            $ref: '#/components/schemas/PullRequestPipelineStage'
+          type: array
+        with:
+          $ref: '#/components/schemas/PullRequestWith'
+      required:
+      - account
+      - date_from
+      - date_to
+      - stages
```

Comment:
If I understood 5bb3f6d2df8d281bbf3b0efa2067f4dc69709a9a `FilterPullRequestsRequeststages::stages` is no longer required.
```suggestion
```